dbms/tests/queries/0_stateless/01051_random_ascii.reference

```diff
@@ -1,2 +1,2 @@
 String
-1000
\ No newline at end of file
+1000
```

Comment:
You can create reference files simply by
`clickhouse-client -n --testmode < query.sql > query.reference`